### PR TITLE
Cleaned up code, Cleaned up ReadMe, etc. 

### PR DESCRIPTION
**Pull request by ledges**
sexy.

### CODE REVIEW

**Comment by lnxcz** [comment]
yo! thanks for the pr, i'll check it out in a bit

File: main.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    )
+                else:
+                    code = "".join(
+                        [
+                            random.choice(string.ascii_letters + string.digits)
+                            for i in range(16)
+                        ]
+                    )
+                req = this.session.get(
+                    f"https://discordapp.com/api/v6/entitlements/gift-codes/{code}",
```
Comment (lnxcz) [inline main.py:62]: i would bump the api version to the latest (9) due to deprecation

File: main.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        code_type = input(
+            f"{Fore.LIGHTMAGENTA_EX}[{strftime('%H:%M', localtime())}] Code Type (boost, classic): "
+        )
+        prox = input(
+            f"{Fore.LIGHTMAGENTA_EX}[{strftime('%H:%M', localtime())}] Proxies (True, False): "
+        )
+        codes = input(
+            f"{Fore.LIGHTMAGENTA_EX}[{strftime('%H:%M', localtime())}] Number of codes: "
+        )
+        SapphireGen(code_type, prox, codes).generate(scrape=False)
```
Comment (lnxcz) [inline main.py:102]: wouldn't it be better if the proxy scrape was part of the input?
like:

```py
scrape_proxy = False

prox = input(
    f"{Fore.LIGHTMAGENTA_EX}[{strftime('%H:%M', localtime())}] Proxies (True, False): "
)
if prox == True:
    scrape_proxy = input(
        f"{Fore.LIGHTMAGENTA_EX}[{strftime('%H:%M', localtime())}] Scrape proxies? (True, False): "
    )


SapphireGen(code_type, prox, codes).generate(scrape=scrape_proxy)
```

File: README.md [hunk 3]
```diff
@@ -32,15 +32,18 @@
...
 
 
 # What's new?
 
-#### -When the user selects "auto-scrape proxy" option, when the proxies are invalids, the program re-scrapes the proxies, and continues the checking.
-#### -Improved the Readme :wow:
+* Massive code cleanup
+* Easily Selectable Options w/ good looking UI.
+
+`To change whether or not to scrape proxies, go to the bottom of the code and at ".generate(scrape=False") just change it to True.`
```
Comment (lnxcz) [inline README.md:46]: don't forget to change if it will be as input

**Review by lnxcz — COMMENTED** [review]
few bits, otherwise lgtm

File: main.py (hunk 2 above)
Comment (ledges) [reply main.py:102]: I did, I must've not have updated the code when I made the pr.

File: README.md (hunk 3 above)
Comment (ledges) [reply README.md:46]: I will go through all these changes give me 10 - 15 minutes!

File: main.py (hunk 1 above)
Comment (ledges) [reply main.py:62]: I could remove /v6, this would by default make it use the current and most intact api version.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
+from time import localtime, strftime, sleep
+from colorama import Fore
+import requests
+import random
+import string
+import time
```
Comment (lnxcz) [inline main.py:6]: unused import
```suggestion

```

File: README.md [hunk 5]
```diff
@@ -32,15 +32,18 @@
...
 
 
 # What's new?
 
-#### -When the user selects "auto-scrape proxy" option, when the proxies are invalids, the program re-scrapes the proxies, and continues the checking.
-#### -Improved the Readme :wow:
+* Massive code cleanup
+* Easily Selectable Options w/ good looking UI.
+
+`To change whether or not to scrape proxies, input True or False when it says "Scrape Proxies"`
```
Comment (lnxcz) [inline README.md:46]: i would just remove this text
```suggestion

```

**Comment by lnxcz** [comment]
alr ty for the pr once again, merging

**Comment by ledges** [comment]
> alr ty for the pr once again, merging

np :)

**Pull request merged**